### PR TITLE
Feral humans blob psychosis

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary

SUMMARY: Content "Feral Humans with blob psychosis"

#### Purpose of change

I'm not sure if there is a specific issue or project related to this but ever since I started playing people have spoken about there being a difference between feral humans and the undead.  I've spoken with Erk and others on this topic at length and decided to start working on adding this.

#### Describe the solution

Add feral human mobs.  Give them some minor ranged attacks to add some variety to hordes.  They butcher as humans, therefore this PR also adjusts "CANNIBAL" to 3 points.  Still need to add spawn groups and possibly a pistol/rifle wielding upgrade.

#### Describe alternatives you've considered

none

#### Testing

TBD

### CODE REVIEW

File: data/json/monsters/feral_humans.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        "fake_per": 6,
+        "require_targeting_player": false,
+        "ranges": [ [ 2, 5, "DEFAULT" ] ],
+        "description": "They throw a loose brick at you!"
+      }
+    ],
+    "death_drops": "feral_human_death_drops",
+    "death_function": [ "NORMAL" ],
+    "anger_triggers": [ "FRIEND_DIED", "FRIEND_ATTACKED", "HURT" ],
+    "flags": [ "SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "BLEED", "HUMAN", "CAN_OPEN_DOORS" ]
```
Comment (LaVeyanFiend) [inline data/json/monsters/feral_humans.json:45]:
```suggestion
    "flags": [ "SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "BLEED", "HUMAN", "CAN_OPEN_DOORS", "PATH_AVOID_DANGER_2" ]
```

**Comment by Xenomorph-III** [comment]
One thing, what will happen to the current feral zombies - as in the "feral runner", "feral hunter" and "feral predator"? Shouldn't these be changed to "zombie runner", "zombie hunter", and "zombie predator"?

**Comment by Maleclypse** [comment]
> One thing, what will happen to the current feral zombies - as in the "feral runner", "feral hunter" and "feral predator"? Shouldn't these be changed to "zombie runner", "zombie hunter", and "zombie predator"?

Done :)

**Comment by GroeneAppel** [comment]
Wouldn't 3 points for Cannibal be a bit too much? Considering it only adds an extra source of (joyful) food. If an increase is needed, an increase to 2 points is probably enough.

**Comment by Maleclypse** [comment]
> Wouldn't 3 points for Cannibal be a bit too much? Considering it only adds an extra source of (joyful) food. If an increase is needed, an increase to 2 points is probably enough.

I'll have to double check but I thought it also added a source of leather and the ability to use human bones in projects and to make charcoal without suffering morale penalties.  It also grants the benefits of the 1 point trait Strict Humanitarian which allows you to butcher mutants and some nethereum creatures that are human-like without morale penalties.  Cannibalism should have been moved to 2 points when Strict Humanitarian was introduced, if we were really keeping track.

**Comment by dissociativity** [comment]
this combined with some feral human tribal camp type deals for maps extras could be interesting, brainstorming some cool future PRs for primitive style feral human variants

**Comment by I-am-Erk** [comment]
The archer is too complex for these ferals. These are really barely more than zombies, they couldn't craft or use a bow.

**Comment by dissociativity** [comment]
they're not basically primitive humans able to make basic tools?

**Comment by I-am-Erk** [comment]
No, these early stage ferals are more like a 28 days later rage zombie, but retaining a little more muscle memory. There will be higher up versions eventually that can use more complex tools, running all the way up to crazy cannibal or psycho NPC's that are hard to distinguish from normal people

**Comment by Maleclypse** [comment]
I added "looks_like": "zombie"  any better suggestions for looks like?

**Comment by dissociativity** [comment]
CHUD, they're underground feral humans already pretty much

**Comment by jeremyshannon** [comment]
> CHUD, they're underground feral humans already pretty much, these are different enough to zombies to not look the same imo.

CHUDs really ought to be fast, terrifying reptilian things like in the first movie, C.H.U.D. II was the one that had "cannibal huamns with blacked out eyes" because it had zero budget and was playing it for laughs.

**Pull request merged**